create_filter_cascade/certs_to_crlite.py

```diff
@@ -350,6 +394,22 @@ def saveMLBF(args, stats, cascade):
     sw.end('save')
 
 
+def find_additions(*, old_by_issuer, new_by_issuer):
+    added = {}
+    for issuer_b64 in new_by_issuer:
```

Comment:
I wonder if we want to verify that anything that was in the old set but isn't in the new set has expired?

Reply:
I like the idea, but this code doesn't have the knowledge to check that. It could emit all the "losses" that are presumably to expirations which could then be checked after the fact -- what do you think?

Reply:
That seems reasonable.